<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>601</width>
      <height>351</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="DetectTab">
     <attribute name="title">
      <string>Detect Mode</string>
     </attribute>
     <widget class="QPushButton" name="CapturePicture">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>160</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>Capture Picture</string>
      </property>
     </widget>
     <widget class="QLabel" name="FilterLabel1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>160</width>
        <height>120</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(233, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="FilterLabel2">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>10</y>
        <width>160</width>
        <height>120</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(233, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="FilterLabel3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>160</width>
        <height>120</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(233, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="FilterLabel4">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>140</y>
        <width>160</width>
        <height>120</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(233, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>10</y>
        <width>240</width>
        <height>180</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(233, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="LoadCapPic">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>270</y>
        <width>160</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>Load</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="CalibrationTab">
     <attribute name="title">
      <string>Calibration Mode</string>
     </attribute>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>10</y>
        <width>371</width>
        <height>311</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_2">
       <widget class="QLabel" name="CalLabel2">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>0</y>
          <width>160</width>
          <height>120</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(233, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="CalLabel4">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>130</y>
          <width>160</width>
          <height>120</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(233, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="CalLabel3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>160</width>
          <height>120</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(233, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="CalLabel1">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>160</width>
          <height>120</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(233, 255, 255);</string>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="LoadCalButtom">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>260</y>
          <width>160</width>
          <height>50</height>
         </rect>
        </property>
        <property name="text">
         <string>Load Calibration Pic From File</string>
        </property>
       </widget>
       <widget class="QPushButton" name="CaptureCalButtom">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>260</y>
          <width>160</width>
          <height>50</height>
         </rect>
        </property>
        <property name="text">
         <string>Capture the Calibration Pictures</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page">
       <widget class="QLabel" name="WarpCalLabel1">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>160</width>
          <height>120</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(233, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="WarpCalLabel2">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>0</y>
          <width>160</width>
          <height>120</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(233, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="WarpCalLabel3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>160</width>
          <height>120</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(233, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="WarpCalLabel4">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>130</y>
          <width>160</width>
          <height>120</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(233, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_3">
       <widget class="QLabel" name="CalResultLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>320</width>
          <height>240</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(233, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QListWidget" name="ProgressList">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>191</width>
        <height>311</height>
       </rect>
      </property>
      <property name="currentRow">
       <number>0</number>
      </property>
      <item>
       <property name="text">
        <string>Capture Calibration Picture</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>See The Warping Result</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>The Stitching Result of the Calibration Photo</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Black Calibration</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>619</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
